index.html

```diff
@@ -0,0 +1,177 @@
+              <li><a class="static-link" href="#">Health campaign</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+
+      <section class="home-section">
+        <div class="home-section_presentation">
+          <div class="intro">
+            <h2 class="hello">"Hello! Fitness Family"</h2>
+            <h1 class="h1">Fit-Fam National Summits(FFNS 2022)</h1>
+          </div>
+
+          <p class="home-paragraph">
+            Benefits of getting involved in physical activities and healthy eating lifestyle.
+          </p>
+          <div class="date">2022.03.11(MON) ~ (FRI)</div>
+          <p class="venue">@DOMINION CITY Ajah-Lagos,Nigeria.</p>
+        </div>
+
+        <!-- Programs -->
+        <article class="home-section_program">
+          <div class="program_h2">
+            <h2 class="main_program">Main Program</h2>
+            <hr class="program_hr" />
+          </div>
+          <div class="program">
+            <img src="./img/lecture.png" alt="lecture-icon" />
+            <h3>Lecture</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto
+              modi velit facilis sit fugit eveniet
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/workshop.png" alt="" />
+            <h3>Workshop</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque
+              doloribus ipsa qui incidunt facere.
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/lecture.png" alt="">
+            <h3>HeartCare</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi
+              veritatis ea aliquid eos inventore?
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/forum.png" alt="health-image" />
+            <h3>Forum</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Laudantium corporis porro dolore mollitia autem.
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/ignite.png" alt="health-image" />
+            <h3>Ignite</h3>
+            <p>
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eius
+              neque impedit expedita est repudiandae.
+            </p>
+          </div>
+          <button type="button" class="program-btn">Join Fitnesss Family-2022</button>
+          <span class="see_program">
+            <a href="#">see the whole program</a>
+          </span>
+        </article>
+
+        <!-- Speakers -->
+        <article class="speakers">
+          <div class="speaker_h2">
+            <h2>Featured Speakers</h2>
+            <hr class="program_hr" />
+          </div>
+
+          <!-- Speakers Details -->
+          <div class="speaker_container"></div>
```

Comment:
- [ ] Kindly, remove the big distance between the title and the speakers:

Your design:
![image](https://user-images.githubusercontent.com/83596135/163587003-6d87ea51-8b18-4eed-9ddc-b42ecfe04453.png)